dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftMetadataConnector.java

```diff
@@ -65,6 +67,13 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
     out.add(new FormatTask(FORMAT_NAME));
     out.add(new RedshiftEnvironmentYamlTask());
 
+    // AWS API tasks, enabled by default if IAM credentials are provided
+    Optional<AWSCredentialsProvider> awsCredentials =
+        AbstractAwsApiTask.createCredentialsProvider(arguments);
+    if (awsCredentials.isPresent()) {
```

Comment:
I would recommend avoiding using `isPresent` followed by `get`. In this case, you can use `ifPresent`:

```
AbstractAwsApiTask.createCredentialsProvider(arguments)
  .ifPresent(awsCredentials ->
    out.add(new RedshiftClusterNodesTask(awsCredentials));
```